Src/ElasticScale.Client/ShardManagement/ShardMap/ShardMapUtils.cs

```diff
@@ -71,5 +92,27 @@ static ShardMapUtils()
                             CultureInfo.InvariantCulture);
             }
         }
+
+        /// <summary>
+        /// Validate .Net runtime 4.6 if Active Directory Aughentication is specified in connection string.
+        /// </summary>
+        /// <param name="connectionString">Connection string to validate.</param>
+        internal static void ValidateAuthenticationInConnectionString(string connectionString)
```

Comment:
Agreed. Will just create SqlConnectionStringBuilder() which will throw if (AAD connection string + not running with .NET 4.6).